packages/protocol/contracts/common/FeeHandlerSeller.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../common/FixidityLib.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "./UsingRegistry.sol";
+import "../common/Initializable.sol";
+
+contract FeeHandlerSeller is Ownable, Initializable, UsingRegistry {
```

Comment:
nit: not sure its a monorepo practice, but in stakedcelo we would document the contract with some overall description.